architecture/mce-overview-ocp.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
+[id="mce-overview-ocp"]
+= About multicluster engine for Kubernetes operator
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: mce-overview-ocp
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator (MCE). The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available in two ways:
+
+* As a standalone operator that you install as part of your OpenShift or Red Hat OpenShift Kubernetes Engine subscription
+* As part of link:https://access.redhat.com/products/red-hat-advanced-cluster-management-for-kubernetes[Red Hat Advanced Cluster Management for Kubernetes]
+
+[id="mce-on-ocp"]
+== Cluster management with multicluster engine on {product-title}
+
+When you enable multicluster engine on {product-title}, you gain the following capabilities:
+
+* xref:control-plane.adoc#hosted-control-planes-overview_control-plane[Hosted control planes], which is based on the HyperShift project. With hosted control planes, you have a centralized control plane for OpenShift clusters so that you can operate clusters in a hyperscale manner.
+* Hive, which provisions self-managed OpenShift clusters to the hub and completes the initial configurations for those clusters.
+* klusterlet agent, which registers managed clusters to the hub.
+* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of OpenShift, such as SNO on bare metal. The Infrastructure Operator includes link:https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.html[zero touch provisioning (ZTP)], which fully automates cluster creation on bare metal and vSphere provisioning with GitOps workflows to manage deployments and configuration changes.
```

Comment:
```suggestion
* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of {product-title}, such as SNO on bare metal. The Infrastructure Operator includes link:https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.html[zero touch provisioning (ZTP)], which fully automates cluster creation on bare metal and vSphere provisioning with GitOps workflows to manage deployments and configuration changes.
```

Should this link be an xref?

Reply:
@bscott-rh  I had the same question, and I initially tried to make it an xref, but I couldn't get it to work. Then, I noticed that I couldn't find that file anywhere in the openshift-docs repo, so I gave up and used a link. I can play around with it again this morning to see if I can get an xref to work.